ConnectorAutocadCivil/ConnectorAutocadCivil/Entry/App.cs

```diff
@@ -166,9 +173,30 @@ public void Create()
       "Check out our tutorials! Opens a page in your web browser"
     );
     RibbonToolTip docsTip = CreateToolTip("Docs", "Check out our documentation! Opens a page in your web browser");
-    RibbonButton community = CreateButton("Community", "SpeckleCommunity", null, helpButton, communityTip, "forum");
-    RibbonButton tutorials = CreateButton("Tutorials", "SpeckleTutorials", null, helpButton, tutorialsTip, "tutorials");
-    RibbonButton docs = CreateButton("Docs", "SpeckleDocs", null, helpButton, docsTip, "docs");
+    RibbonButton community = CreateButton(
```

Comment:
Actually I would remove the tutorials and docs links and buttons entirely - we are moving towards deprecating all v2 support. And we don't have updated links for these in V3 yet

Can leave community for help requests

Reply:
I think we should wait until we know what's happening with v2 docs post v3 docs site